compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -1165,11 +1165,12 @@ fn describe_codegen_flags() {
 
 fn print_flag_list<T>(
     cmdline_opt: &str,
-    flag_list: &[(&'static str, T, &'static str, &'static str)],
+    flag_list: &[(&'static str, T, &'static str, &'static str, Option<OptionsTargetModifiers>)],
```

Comment:
Problem: this is outdated, there should now be a `OptionDesc` struct instead of this 5-tuple.

Reply:
Fixed.